apps/events/dashboard/forms.py

```diff
@@ -94,6 +94,12 @@ class Meta:
 
 
 class CreateFeedbackRelationForm(forms.ModelForm, HTML5RequiredMixin):
+
+    def __init__(self, *args, **kwargs):
+        super(CreateFeedbackRelationForm, self).__init__(*args, **kwargs)
+        feedback = Feedback.objects.filter(available=True)
+        self.fields['feedback'].queryset = feedback
```

Comment:
I'm not sure if this is the cleanest way to do this, but hey, if it works \o/

Reply:
It may not look nice, but from my limited time researching the topic it seems this is the way it is done